<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>671</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="mainLayout">
     <item>
      <layout class="QVBoxLayout" name="categoriesLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QListView" name="listView"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="tasksLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item>
        <widget class="QListView" name="listView_2"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QListView" name="listView_3"/>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string> Help</string>
    </property>
    <addaction name="actionSourceCodeOnGitHub"/>
    <addaction name="actionLatestReleasesOnGitHub"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuSortBy">
     <property name="title">
      <string>Sort by</string>
     </property>
     <addaction name="actionTitle"/>
     <addaction name="actionPublicationDate"/>
     <addaction name="actionFinishDate"/>
    </widget>
    <widget class="QMenu" name="menuSortOrder">
     <property name="title">
      <string>Sort order</string>
     </property>
     <addaction name="actionAscending"/>
     <addaction name="actionDescending"/>
    </widget>
    <addaction name="actionShowCompletedTasks"/>
    <addaction name="separator"/>
    <addaction name="menuSortBy"/>
    <addaction name="menuSortOrder"/>
   </widget>
   <addaction name="menuDatabase"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionSeparator">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionSourceCodeOnGitHub">
   <property name="text">
    <string>Windos source code on GitHub</string>
   </property>
  </action>
  <action name="actionLatestReleasesOnGitHub">
   <property name="text">
    <string>Windos latest releases on GitHub</string>
   </property>
  </action>
  <action name="actionShowCompletedTasks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show completed tasks</string>
   </property>
  </action>
  <action name="actionTitle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </action>
  <action name="actionPublicationDate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Publication date</string>
   </property>
  </action>
  <action name="actionFinishDate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Finish date</string>
   </property>
  </action>
  <action name="actionAscending">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ascending</string>
   </property>
  </action>
  <action name="actionDescending">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Descending</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
